engine.py

```diff
@@ -162,7 +141,7 @@ def refresh_engine(engine_name, prev_context, menu_name):
         return
 
     # Get the path of the current open Maya scene file.
-    new_path = maya_scene_path()
+    new_path = MayaEngine.maya_scene_path()
```

Comment:
The name of this class is `SceneEventWatcher`, not `MayaEngine`.

Reply:
I think this is correct? I'm calling the static method `maya_scene_path()` which is in the `MayaEngine` class.
`refresh_engine` isn't actually in the `SceneEventWatcher` class in any case, though I can see the confusion at a glance, it the first method after that class ends.